### PR TITLE
Copy edits

### PR DESCRIPTION
**Pull request by chasballew**
@samyount @gib - Please check for typos, comment on lines, and merge if good.

### CODE REVIEW

File: faq/index.hbs [hunk 1]
```diff
@@ -10,32 +10,123 @@ faq:
...
+        answer: |
+          A container is a lightweight virtual machine. Aptible uses Docker to build and run containers. Standard containers are allocated 1 GB of RAM. Higher RAM configurations are available upon request.
+
+      - question: Disk
+        answer: |
+          Disk storage refers to the underlying volumes used for database storage. Disk storage is enrypted at the filesystem level with 192-bit AES encryption. Aptible manages the encryption keys for you. Disk pricing includes nightly backups for 90 days and monthly backups for 6 years. Backups are stored in a geographic region separate from where your database runs.
+
+      - question: Domain
+        answer: |
+          A domain, or "VHOST," is an SSL/TLS endpoint. Domains are attached to a service and consist of a DNS service, an AWS Elastic Load Balancer, and an NGiNX container.
```
Comment (fancyremarker) [inline faq/index.hbs]: Technically, a set of NGiNX containers. Describing the exact components of a VHOST may be too detailed though.

File: faq/index.hbs [hunk 2]
```diff
@@ -10,32 +10,123 @@ faq:
...
+
+          "Internal" VHOSTs can be used to expose a private service to other services on the same stack. Internal VHOSTs receive private IP addresses and cannot be accessed from the Internet.
+
+      - question: Service
+        answer: |
+          A service is comprised of one or more containers running the same process. By adding containers to a service, you can easily and quickly scale. Services run in your stack's app layer (see our [Reference Architecture Diagram](/pages/assets/aptible-reference-architecture.pdf)).
+
+      - question: Environment
+        answer: |
+          Environments are used to manage permissions. PHI-ready environments run on dedicated stacks. Dev environments run on shared stacks and must not handle PHI. Multiple environments on the same stack do not provide network isolation.
```
Comment (fancyremarker) [inline faq/index.hbs]: I would say: "Environments are used to manage permissions, and to organize apps and databases."

File: faq/index.hbs [hunk 3]
```diff
@@ -10,32 +10,123 @@ faq:
...
+        answer: |
+          A service is comprised of one or more containers running the same process. By adding containers to a service, you can easily and quickly scale. Services run in your stack's app layer (see our [Reference Architecture Diagram](/pages/assets/aptible-reference-architecture.pdf)).
+
+      - question: Environment
+        answer: |
+          Environments are used to manage permissions. PHI-ready environments run on dedicated stacks. Dev environments run on shared stacks and must not handle PHI. Multiple environments on the same stack do not provide network isolation. 
+
+      - question: Stack
+        answer: |
+          The underlying infrastructure we use to build your environments. Each stack consists of an AWS VPC, EC2 instances, Elastic Load Balancers, Elastic Block Store volumes, and the Aptible platform utilities. Stacks provide network isolation.
```
Comment (fancyremarker) [inline faq/index.hbs]: Perhaps EBS instead of "Elastic Block Store," for consistency? No strong opinion.

File: faq/index.hbs [hunk 4]
```diff
@@ -10,32 +10,123 @@ faq:
...
+      - question: Do I have to sign a long-term contract?
+        answer: |
+          No. Platform Account customers can choose a month-to-month option for $799/mo base. Most customers sign up for a 90-day trial at $499/mo base and continue on annual contracts at that base price.
+
+  - section: Platform
+    section_slug: platform
+    questions:
+      - question: What technology can I use with Aptible?
+        answer: |
+          Any Linux-based language or framework. We have  [Quickstart Guides](https://support.aptible.com/quickstart) for many popular languages and databases. No Windows, sorry.
```
Comment (fancyremarker) [inline faq/index.hbs]: Minor (doesn't affect rendering): two spaces instead of one.

Comment (fancyremarker) [inline faq/index.hbs]: :+1: :100: for no Windows support.

**Comment by fancyremarker** [comment]
This is really good. Thanks @chasballew .

**Comment by gib** [comment]
Let's roll these out with the pricing calc.

**Pull request merged**